app/controller/requests.rb

```diff
@@ -9,6 +11,10 @@ class Bnb < Sinatra::Base
       booking = Booking.get(params[:host_request_id])
       booking.status = 'confirmed'
       booking.save
+
+      space = Space.get(booking.space_id)
```

Comment:
this var is not used or modified, so no need to call it and then save the db entry